pkg/config/config.go

```diff
@@ -16,6 +16,12 @@ const (
 	configFileName = "odo"
 )
 
+// OdoSettings holds all odo specific configurations
+type OdoSettings struct {
+	// holds the status for update checker
+	UpdateNotification *bool `json:"updatenotification,omitempty"`
```

Comment:
This variable shouldn't be exposed (lowercase first letter). Only `GetUpdateNotification` function has to be usd to read it, to ensure correct value.

Reply:
capitalizing this variable since its a JSON tag